contracts/axelar-gas-service/tests/test.rs

```diff
@@ -160,7 +201,11 @@ fn add_gas_fails_with_zero_gas_amount() {
     let sender: Address = Address::generate(&env);
     let message_id = message_id(&env);
     let gas_amount: i128 = 0;
-    let token = setup_gas_token(&env, gas_amount);
+    let asset = env.register_stellar_asset_contract_v2(Address::generate(&env));
+    let token = Token {
+        address: asset.address(),
```

Comment:
```suggestion
        address: Address::generate(),
```
maybe this is sufficient?

Reply:
that's true, updated